evm.md

```diff
@@ -1881,6 +1914,36 @@ In the YellowPaper, each opcode is defined to consume zero gas unless specified
     rule [#memoryUsageUpdate.some]: #memoryUsageUpdate(MU, START, WIDTH) => maxInt(MU, (START +Int WIDTH) up/Int 32) requires WIDTH  >Int 0
 ```
 
+Access List Gas
+---------------
+
+```k
+    syntax Bool ::= #usesAccessList ( OpCode ) [function, functional]
+ // -----------------------------------------------------------------
+    rule #usesAccessList(OP) => isAddr1Op(OP) orBool isAddr2Op(OP) orBool OP ==K SLOAD orBool OP ==K SSTORE
+
+    syntax InternalOp ::= "#access" "[" OpCode "]"
+ // --------------------------------------------
+    rule <k> #access [ OP ] => #gasAccess(SCHED, OP) ~> #deductGas ... </k>
+         <schedule> SCHED </schedule>
+      requires Ghasaccesslist << SCHED >>
```

Comment:
We don't need `Ghasaccesslist` check here anymore, if you don't want, because we are already checking it above.

So you can have `#gasAccess` have an `[owise]` rule (which will never fire) that goes to `0`, similar to how `#memory` works above.